.github/workflows/az-e2e.yaml

```diff
@@ -0,0 +1,34 @@
+name: azure e2e tests
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator azure e2e tests
+    runs-on: az-ubuntu-2204
+
+    strategy:
+      matrix:
+        runtimeclass: ["kata-qemu", "kata-clh"]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install deps
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y ansible python-is-python3
+
+      - name: Run e2e tests
+        continue-on-error: false
+        run: |
+          cd tests/e2e
+          export PATH="$PATH:/usr/local/bin"
+          ./run-local.sh -r "$RUNTIME_CLASS" -u
+        env:
+          RUNTIME_CLASS: ${{ matrix.runtimeclass }}
```

Comment:
I don't think it should export RUNTIME_CLASS, the -r parameter to run-local.sh should account for that.